config/initializers/secret_token.rb

```diff
@@ -4,4 +4,6 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
+# rubocop:disable Metrics/LineLength
 Safecast::Application.config.secret_token = '39c00ee18c6a2edd472f8f8b5954b6a332636e32b4c80ed9abc9053cc33bf7f1913d4cb6267ea741d10b48acfd9b3b3df2336fd02de83a4290509bc5d97c0edd'
+# rubocop:enable Metrics/LineLength
```

Comment:
Ya, 80 is probably too strict. maybe, somewhere around 100 - 128. I will change the value once all rubocop checks are enabled.